core.js

```diff
@@ -581,7 +581,9 @@ async function _fromTokenizer(tokenizer) {
 		};
 	}
 
-	if (check([0x49, 0x49, 0x2A, 0x0] || [0x4D, 0x4D, 0x0, 0x2A])) {
+	// TIFF-compatible formats can be encoded two ways but little-endian (0x49) is most common.
+	// Big-endian TIFFs are handled below this block.
```

Comment:
This is little-endian I believe [ref](http://fileformats.archiveteam.org/wiki/TIFF#File_signatures).

Please add the `TIFF big-endian` comment to the other signature.

Reply:
Yep this is little-endian, sorry if the comment was confusing!